### PR TITLE
Added support for php

### PR DESCRIPTION
**Pull request by SlavaRa**
@elsassph please review this PR and merge if all ok

### CODE REVIEW

File: test/massive/munit/TestClassHelperTest.hx [hunk 1]
```diff
@@ -35,32 +35,33 @@ class TestClassHelperTest
...
-	public function testConstructor():Void
+	public function testConstructor()
 	{
 		var helper:TestClassHelper = new TestClassHelper(TestClassStub);
 		
 		Assert.isNotNull(helper.test);
 		Assert.isType(helper.test, TestClassStub);
 		Assert.areEqual(helper.type, TestClassStub);
-		
+		#if !php
```
Comment (elsassph) [inline test/massive/munit/TestClassHelperTest.hx]: Is there a PHP target restriction?

Comment (SlavaRa) [reply test/massive/munit/TestClassHelperTest.hx]: now this is a forced change, it will be fixed in the PR for HashLink

**Comment by SlavaRa** [comment]
bump

**Comment by SlavaRa** [comment]
:(

**Comment by SlavaRa** [comment]
bump

**Comment by SlavaRa** [comment]
@elsassph, is there the slightest chance, that there will be a merger?

**Comment by SlavaRa** [comment]
:(

**Comment by SlavaRa** [comment]
@elsassph Thanks!

**Pull request merged**